### ISSUE TITLE
Setting null_value on an integer field causes an exception

### ISSUE THREAD

**Issue by laaksomavrick** [strapdata/elassandra#296]
Hello, I am seeing an exception when I attempt to create a mapping for a UDT with the following attributes. Removing the `null_value` from the `foo` property fixes the issue, leading me to believe there is some issue with `null_value` and integers.

```
"something": {
            "type": "nested",
            "cql_udt_name": "something_udt",
            "cql_collection": "singleton",
            "properties": {
                "foo": {
                    "type": "integer",
                    "null_value": "0",
                    "cql_collection": "singleton",
                    "fields": {
                        "raw": {
                            "type": "integer",
                            "null_value": "0"
                        }
                    }
                },
                "bar": {
                    "type": "keyword",
                    "null_value": "NULL",
                    "cql_collection": "singleton",
                    "normalizer": "custom_normalizer",
                    "fields": {
                        "raw": {
                            "type": "keyword",
                            "null_value": "NULL"
                        }
                    }
                },
                "baz": {
                    "type": "keyword",
                    "null_value": "NULL",
                    "cql_collection": "singleton",
                    "normalizer": "custom_normalizer",
                    "fields": {
                        "raw": {
                            "type": "keyword",
                            "null_value": "NULL"
                        }
                    }
                },
                "value": {
                    "type": "float",
                    "null_value": "0",
                    "cql_collection": "singleton",
                    "fields": {
                        "raw": {
                            "type": "float",
                            "null_value": "0"
                        }
                    }
                }
            }
        },
```

The exception:

```
java.lang.NullPointerException: null
        at org.elasticsearch.index.mapper.NumberFieldMapper$NumberType$6.createFields(NumberFieldMapper.java:696)
        at org.elasticsearch.index.mapper.NumberFieldMapper$NumberType$4.createFields(NumberFieldMapper.java:516)
        at org.elasticsearch.index.mapper.NumberFieldMapper.createField(NumberFieldMapper.java:1151)
        at org.elassandra.index.ElasticSecondaryIndex.addField(ElasticSecondaryIndex.java:390)
        at org.elassandra.index.ElasticSecondaryIndex.addField(ElasticSecondaryIndex.java:485)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$RowcumentIndexer$Rowcument.buildContext(ElasticSecondaryIndex.java:2074)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$WideRowcumentIndexer$WideRowcument.buildContext(ElasticSecondaryIndex.java:1421)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$RowcumentIndexer$Rowcument.index(ElasticSecondaryIndex.java:2160)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$RowcumentIndexer$Rowcument.index(ElasticSecondaryIndex.java:2144)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$WideRowcumentIndexer$WideRowcument.index(ElasticSecondaryIndex.java:1421)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$RowcumentIndexer$Rowcument.write(ElasticSecondaryIndex.java:2127)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$WideRowcumentIndexer$WideRowcument.write(ElasticSecondaryIndex.java:1421)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$WideRowcumentIndexer.readBeforeWrite(ElasticSecondaryIndex.java:1474)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$WideRowcumentIndexer.update(ElasticSecondaryIndex.java:1542)
        at org.elassandra.index.ElasticSecondaryIndex$ImmutableMappingInfo$RowcumentIndexer.finish(ElasticSecondaryIndex.java:1845)
        at java.lang.Iterable.forEach(Iterable.java:75)
        at org.apache.cassandra.index.SecondaryIndexManager.indexPartition(SecondaryIndexManager.java:642)
        at org.apache.cassandra.index.internal.CollatedViewIndexBuilder.build(CollatedViewIndexBuilder.java:139)
        at org.apache.cassandra.db.compaction.CompactionManager$14.run(CompactionManager.java:1637)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at org.apache.cassandra.concurrent.NamedThreadFactory.lambda$threadLocalDeallocator$0(NamedThreadFactory.java:81)
        at java.lang.Thread.run(Thread.java:748)
```

Let me know if you need any information from me. Thanks.

**Comment by vroyer** [comment]
Yes there is an issue for numbers, fixed in next release 6.2.3.17
Thanks for reporting it.

**Comment by laaksomavrick** [comment]
Thanks @vroyer, this fix will help me out a lot. Cheers for taking the time.
